### PR TITLE
New action to build the FE

### PR DESCRIPTION
**Pull request by macaab26**
## What's changing

I'm adding a new action to build the FE

## How to test it

See the execution of the action

## I already...

- [ ] added some tests for any new functionality
- [ ] updated the documentation
- [ ] checked if a (backend) DB migration step was required and included it if required

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -30,7 +30,7 @@ local-logs:
 
 # Launches lumigator in 'user-local' mode (All services running locally, using latest docker container, no code mounted in)
 start-lumigator: .env
-	RAY_ARCH_SUFFIX=$(RAY_ARCH_SUFFIX) docker compose --profile local -f $(LOCAL_DOCKERCOMPOSE_FILE) up -d
+	RAY_ARCH_SUFFIX=$(RAY_ARCH_SUFFIX) docker compose --profile local-fe -f $(LOCAL_DOCKERCOMPOSE_FILE) up -d
```
Comment (agpituk) [inline Makefile:33]: I would move this to a different PR. Approved once that's done

Comment (peteski22) [reply Makefile:33]: https://github.com/mozilla-ai/lumigator/pull/452

**Review by peteski22 — APPROVED** [review]
